README.md

```diff
@@ -156,6 +156,9 @@ running on `127.0.0.1:8081`:
 RUST_LOG=xaynet=info cargo run --bin test-drive-net -- -n 20 -u http://127.0.0.1:8081
 ```
 
+For more in-depth details on how to run examples, see the accompanying Getting
+Started guide under [rust/src/examples.rs](./rust/src/examples.rs).
```

Comment:
Why putting that in `src/`? We could just add this to the examples themself in `rust/examples`.

Reply:
that was my original thought too, but it seems `cargo doc` doesn't render docs under `examples` (I saw an open issue for addressing that). So instead, it's a module here which is just pure-documentation, which does at least get rendered along with the rest. I actually got this idea from the [`csv`](https://docs.rs/csv/1.0.0-beta.5/csv/index.html) crate (which has pure-documentation modules `cookbook` and `tutorial`).